app/views/poms/_poms_table.html.erb

```diff
@@ -1,31 +1,30 @@
-<table class="govuk-table responsive tablesorter">
+<table class="govuk-table responsive">
+  <%= case_mix_key %>
   <thead class="govuk-table__head">
   <tr class="govuk-table__row">
-    <th class="govuk-table__header" scope="col">POM name</th>
-    <th class="govuk-table__header" scope="col">Tier A<br/>cases</th>
-    <th class="govuk-table__header" scope="col">Tier B<br/>cases</th>
-    <th class="govuk-table__header" scope="col">Tier C<br/>cases</th>
-    <th class="govuk-table__header" scope="col">Tier D<br/>cases</th>
-    <th class="govuk-table__header" scope="col">No tier<br/>cases</th>
-    <th class="govuk-table__header" scope="col">Total</th>
-    <th class="govuk-table__header header-width__8" scope="col">Working<br/>pattern</th>
+    <th class="govuk-table__header govuk-!-width-one-quarter" scope="col">POM</th>
+    <th class="govuk-table__header" scope="col">Working pattern</th>
     <th class="govuk-table__header" scope="col">Status</th>
-    <th class="govuk-table__header sorter-false" scope="col">Action</th>
+    <th class="govuk-table__header govuk-!-width-one-quarter" scope="col">Case mix by tier</th>
+    <% if @prison.womens? %>
+      <th class="govuk-table__header govuk-table__header--numeric" scope="col">High complexity cases</th>
+    <% end %>
+    <th class="govuk-table__header govuk-table__header--numeric" scope="col">Total cases</th>
   </tr>
   </thead>
   <tbody class="govuk-table__body">
   <% poms.each_with_index do |pom, i| %>
     <tr class="govuk-table__row <%= pom_type %>_pom_row_<%= i %>">
-      <td aria-label="POM name" class="govuk-table__cell "><%= full_name(pom) %></td>
-      <td aria-label="Tier A cases" class="govuk-table__cell "><%= pom.tier_a %></td>
-      <td aria-label="Tier B cases" class="govuk-table__cell "><%= pom.tier_b %></td>
-      <td aria-label="Tier C cases" class="govuk-table__cell "><%= pom.tier_c %></td>
-      <td aria-label="Tier D cases" class="govuk-table__cell "><%= pom.tier_d %></td>
-      <td aria-label="No tier cases" class="govuk-table__cell "><%= pom.no_tier %></td>
-      <td aria-label="Total" class="govuk-table__cell "><%= pom.total_cases %></td>
+      <td aria-label="POM" class="govuk-table__cell"><%= link_to full_name_ordered(pom), prison_pom_path(@prison.code, nomis_staff_id: pom.staff_id) %></td>
       <td aria-label="Working pattern" class="govuk-table__cell"><%= format_working_pattern(pom.working_pattern) %></td>
-      <td aria-label="Status" class="govuk-table__cell "><%= pom.status.capitalize %></td>
-      <td aria-label="Action" class="govuk-table__cell "><%= link_to 'View', prison_pom_path(@prison.code, nomis_staff_id: pom.staff_id) %></td>
+      <td aria-label="Status" class="govuk-table__cell"><%= status(pom).capitalize %></td>
+      <td aria-label="Case mix by tier" class="govuk-table__cell"><%= case_mix_bar(pom) %></td>
```

Comment:
Likewise, the `case_mix_bar` helper method could perform its own counts rather than depending on `PomWrapper#tier_a`, `PomWrapper#tier_b`, and so on...

```ruby
pom.count_offenders { |o| o.tier == 'A' }
# and so on...
```